developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
+
+### Publish the module
+
+1. In your `aptos-core` source checkout, navigate to the `aptos-core/aptos-move/move-examples/mint_nft/1-Create-NFT` directory:
+
 ```shell
-aptos init --profile nft-receiver
+cd aptos-move/move-examples/mint_nft/1-Create-NFT
 ```
 
-1. When prompted, select `testnet` by entering it:
+2. Run `aptos move publish` to publish the Move module in that directory, updating it with your default account address from `.aptos/config.yaml`:
 
 ```shell
-Configuring for profile nft-receiver
-Choose network from [devnet, testnet, mainnet, local, custom | defaults to devnet]
-testnet
+aptos move publish --named-addresses mint_nft=<default-account-address>
 ```
 
-1. When prompted for your private key, hit enter to generate a new key:
+For example:
+
 ```shell
-Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
+aptos move publish --named-addresses mint_nft=a911e7374107ad434bbc5369289cf5855c3b1a2938a6bfce0776c1d296271cde
 ```
 
-1. Receive output resembling:
+3. Receive output asking you to accept a gas fee for the transaction:
+
 ```shell
-No key given, generating key...
-Account blah does not exist, you will need to create and fun the account through a community faucet e.g. https://aptoslabs.com/testnet-faucet, or by transferring funds from another account
+    Compiling, may take a little while to download git dependencies...
+    INCLUDING DEPENDENCY AptosFramework
+    INCLUDING DEPENDENCY AptosStdlib
+    INCLUDING DEPENDENCY AptosToken
+    INCLUDING DEPENDENCY MoveStdlib
+    BUILDING Examples
+    package size 2770 bytes
+    Do you want to submit a transaction for a range of [1164400 - 1746600] Octas at a gas unit price of 100 Octas? [yes/no]
+```
 
----
-Aptos CLI is now set up for account blah as profile nft-receiver!  Run `aptos --help` for more information about commands
-{
-  "Result": "Success"
-}
-➜  devel
+4. Enter `yes` and receive results indicating success:
+
+```shell
+    {
+      "Result": {
+        "transaction_hash": "0x576a2e9481e71b629335b98ea75c87d124e1b435e843e7a2ef8938ae21bebfa3",
+        "gas_used": 11679,
+        "gas_unit_price": 100,  
+        "sender": "a911e7374107ad434bbc5369289cf5855c3b1a2938a6bfce0776c1d296271cde",
+        "sequence_number": 0,
+        "success": true,
+        "timestamp_us": 1669659103283876,
+        "version": 12735152,
+        "vm_status": "Executed successfully"
+      }
+    }
+```
+
+### See the module on Aptos Explorer
+
+1. Go to the [Aptos Explorer](https://explorer.aptoslabs.com/) in a web browser>
+2. Select the network you used: testnet
+3. Search for the transaction by `transaction_hash` in the search field, using your own unique transaction hash.
+4. View the changes made in publishing this module under the *Changes* tab.
+
+## Mint a token to a receiver
```

Comment:
so for `1-Create-NFT/sources/create_nft.move`, we actually are not using a resource account. we were trying to make a point for the usage of the resource account. since if we use a normal account to publish the package, we would need two signer to sign the `mint_event_ticket` transaction, which is impractical in production

Reply:
I found this confusing too. We don't use a resource account at the CLI, but we do start to describe minting an NFT to the receiver at: 
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/1-Create-NFT/sources/create_nft.move#L131

So I felt I needed to point that out. I can easily remove that reference though. The next section (#2) covers use of a resource account anyway. Please advise. Thanks!

Reply:
I moved this content to part 2 on resource accounts. Thanks!